org.js.nuclear.Nuclear.json

```diff
@@ -2,9 +2,8 @@
     "app-id": "org.js.nuclear.Nuclear",
     "runtime": "org.freedesktop.Platform",
     "runtime-version": "19.08",
-    "branch": "19.08",
+    "branch": "0.6.3",
     "sdk": "org.freedesktop.Sdk",
-    "base": "io.atom.electron.BaseApp",
```

Comment:
why did you remove this if this is an electron app?

Reply:
i removed it as it now use pre-compiled binaries  that contain all the required  things such as the electron binary. this makes the electron base app increase the size of the final product.

Reply:
ah I see. I missed that. But maybe that how instead a real build could be done for non x86_64 architecture, in the future.